docs/blog/2018-10-16-unstructured-data/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Using unstructured data in Gatsby
+date: 2018-10-16
+author: Amberley Romo
+tags: ["sourcing", "data"]
+---
+
+When creating a experiences on the web, one of the questions you can't get around asking (unless you're writing hard-coded HTML) is: how do I get my data from point A (wherever it lives) to point B (where I want it). When wanting to take advantage of modern tooling like React, webpack, babel, etc., this can end up being a deceptively complex question.
```

Comment:
What about, instead of a colon, using quotation marks? When creating...is, "how do I get my data from point A to point B?".

That way you can use a question mark at the end of the question without turning the whole sentence into a question!

P.S. Colon thoughts (haha that sounds funny): I feel ambivalent about colon rules in most cases, so just in case this is useful for future reference, colons are "supposed" to come after a full sentence. I found that out two years ago and realized I had never used them after a full sentence my whole life!